packages/forms-web-app/src/pages/process-guide/_includes/process-guide-steps.njk

```diff
@@ -5,98 +5,105 @@
     steps: [
       {
           heading: {
-            text: processGuide.steps.preApplication.title
+            text: t('processGuide.preApplication.subHeading')
```

Comment:
I think these translations should be set in the 'get-process-guide-steps.js' middleware function as we will also need the active step translations for the process guide pages.